src/lib.rs

```diff
@@ -151,27 +151,25 @@
 //! [`Sprite`]: https://docs.rs/bevy/0.8.0/bevy/sprite/struct.Sprite.html
 //! [`Transform`]: https://docs.rs/bevy/0.8.0/bevy/transform/components/struct.Transform.html
 
-use bevy::prelude::*;
 use std::time::Duration;
 
 #[cfg(feature = "bevy_asset")]
 use bevy::asset::Asset;
-
+use bevy::prelude::*;
 use interpolation::Ease as IEase;
 pub use interpolation::{EaseFunction, Lerp};
 
-pub mod lens;
```

Comment:
Nope, this is just my auto formatter. Happy to those up.

Reply:
Any idea why it's doing that? If there's a best practice or Rust guideline I'm happy to follow it. I'm all in favor of all those things being dictated by formatting tools rather than humans, that makes files consistent and avoid fights on code.style 🥲

Reply:
Looks like other people complained but haven't received a response so no clue lol: https://github.com/intellij-rust/intellij-rust/issues/2259#issuecomment-455579097